rotkehlchen/db/upgrades/v32_v33.py

```diff
@@ -72,6 +72,10 @@ def _create_new_tables(cursor: 'Cursor') -> None:
 """)
 
 
+def _refactor_blockchain_account_labels(cursor: 'Cursor') -> None:
```

Comment:
Decided to go with a db upgrade since blockchain accounts functionality (e.g. API, Database, etc.) already fully support `None` label